src/renderer/domains/collectives/referendumMeta/model.ts

```diff
@@ -42,7 +45,8 @@ const {
             title: x.title,
             description: x.content,
             track: x.track,
-            createdAt: x.indexer.blockTime,
+            status: x.state.name,
```

Comment:
Using `x` all the time is super awkward 😕
```suggestion
            status: p.state.name,
```

Reply:
Why? It's common shorthand for small mapping functions